lib/matplotlib/axes/_axes.py

```diff
@@ -3427,13 +3427,17 @@ def extract_err(name, err, data, lolims, uplims):
                 the note in the main docstring about this parameter's name.
             """
             try:
-                low, high = np.broadcast_to(err, (2, len(data)))
+                np.broadcast_to(err, (2, len(data)))
```

Comment:
Is it possible to check for this directly, rather than via an error?  Maybe this is just as efficient?

Reply:
I can't find anything under https://numpy.org/doc/stable/reference/routines.array-manipulation.html#changing-number-of-dimensions :(

Reply:
... I guess I just mean `int(len(err)/2)==len(err)/2`, or whatever is more elegant than that?

Reply:
Performance should not matter. This should be proportional to len(data) in computation/mempry and is only called <s>once</s> twice for the whole errorbar function.